pages/vi/gitrepositories.md

```diff
@@ -12,8 +12,59 @@ On GitHub, our software code is organized in repositories, each of which represe
 
 You previously forked a repository from treehouses to work on, which means that that a version of the treehouses repository was available on your own GitHub account. You were then able to modify your fork, then send an update to the upstream repository in the form of a pull request. 
 
+## Summary of Steps
+Generally, this is what the structure of your workflow will look like: 
+
+![Image](images/upstream_diagram.png)
```

Comment:
@sydneyaldo think this file is missing

Reply:
The picture file is missing?

Reply:
All I did was change the position, it shouldn't be missing

Reply:
oops sorry you are right ;)